prime-router/src/main/kotlin/Translator.kt

```diff
@@ -133,16 +133,20 @@ class Translator(private val metadata: Metadata, private val settings: SettingsP
         )
         if (routingFilteredReport.isEmpty()) return routingFilteredReport
 
-        // Do conditionFiltering on the routingFilteredReport
-        val conditionFilteredReport = filterByOneFilterType(
-            routingFilteredReport,
-            receiver,
-            organization,
-            ReportStreamFilterType.CONDITION_FILTER,
-            trackingElement,
-            doLogging = true
-        )
-        if (conditionFilteredReport.isEmpty()) return conditionFilteredReport
+        var conditionFilteredReport = routingFilteredReport
+        // condition filter can only be used on UP
+        if (receiver.topic == Topic.FULL_ELR || receiver.topic == Topic.FULL_ELR_TEST) {
```

Comment:
Remove this IF. This code is only run for FULL_ELR as this is part of the UP

Reply:
Without this, the error that James brought up `13:24:18.629 [pool-2-thread-32][53] ERROR gov.cdc.prime.router.Translator - NOT CONDITION_FILTER filtering for topic COVID_19. No filters found. ` Continues to show up in the logs because this code does get hit when I run something in  the COVID pipeline.

Reply:
This does not sound right. There should be no data coming through here that is for the UP pipeline.